### ISSUE TITLE
DOC Use BibTeX for citations

### ISSUE THREAD

**Issue by riedgar-ms** [fairlearn/fairlearn#936]
#### Describe the issue linked to the documentation

When we wish to cite papers in our documentation, we are currently writing out the citations manually. Not only is this tedious, but it is also highly error prone (since the links in the text are numbers along the lines of `[4]_`). This is particularly acute if merge conflicts occur in the 'References' section, since then the whole page must be cross checked, to ensure that all the links to the references point at the right paper.

#### Suggest a potential alternative/fix

Somewhat over 30 years ago, BibTex was invented to help with precisely this issue. Furthermore, online paper databases often provide BibTeX snippets for inclusion in `.bib` files. There is also a [BibTeX package for Sphinx](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/quickstart.html) which looks to be under active development. A possible solution suggests itself.

**Comment by iofall** [comment]
I can help with this.

**Comment by hildeweerts** [comment]
Awesome, @iofall! I have assigned the issue to you. Please let us know if you have any questions.

**Comment by romanlutz** [comment]
I believe the PR addressed only one file, not the entire documentation, so I'm reopening it. Please close if I'm wrong.

**Comment by iofall** [comment]
Yes @romanlutz, we only worked on one file and added the dependencies for the bibtex extension. I was thinking about opening issues for the other files - these issues are simple enough for beginners, they just have to go through the `.rst` files, look for citations, add them to the `refs.bib` file and use the `footcite` command with the keys.

If we are planning to have a sprint sometime in the near future, these could be helpful. Let me know what you think otherwise I am happy to work on modifying the other files in the documentation as well.

**Comment by romanlutz** [comment]
I could see it work out well that way, too. I always like things being consistent but then that's a very perfectionist attitude that's not always feasible. At least we should have a list of all the cases (either in this issue or a new issue per file. And we should have a short documentation section explaining how to cite perhaps?

If you're interested I'd say go for it and replace the others. If you prefer doing other tasks by all means leave it to new contributors. I agree it's a fine first issue to work on and familiarize oneself with GitHub.

**Comment by iofall** [comment]
> If you're interested I'd say go for it and replace the others. If you prefer doing other tasks by all means leave it to new contributors. I agree it's a fine first issue to work on and familiarize oneself with GitHub.

Sorry for the late reply. Okay, I could help with more files.

> short documentation section explaining how to cite perhaps?

Yes that is definitely a good idea. Which section would be right for this addition? (I think we could add it to [Contributing Documentation](https://fairlearn.org/v0.7.0/contributor_guide/contributing_documentation.html) section)

**Comment by romanlutz** [comment]
That's the perfect location for adding the documentation! Thanks @iofall .
